main.go

```diff
@@ -165,10 +165,6 @@ func Execute(cmd *cobra.Command, args []string) {
 		ValidatorHandler(w, r, grpcConn)
 	})
 
-	http.HandleFunc("/metrics/validators", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
I'm not sure why we remove it?

Reply:
Do we need to know all validators' status in the chain?

Reply:
If we don't call this endpoint, there's no need in removing it.

Reply:
fixed